scalableminds/webknossos
# Include result link in job slack notification

The result link was hitherto constructed in the front-end. Now I construct it in the back-end instead and send it both to the front-end and to the slack notification.

### Steps to test:
- set up jobs locally
- run jobs of different type
- result links in jobs page should still work
- job slack notifications should include link (set slackNotifications.verboseLoggingEnabled = true to test)

### Issues:
- fixes #5857 

------
